pandevice/base.py

```diff
@@ -2775,10 +2781,10 @@ class XapiWrapper(pan.xapi.PanXapi):
         def __init__(self, *args, **kwargs):
             self.pan_device = kwargs.pop('pan_device', None)
             pan.xapi.PanXapi.__init__(self, *args, **kwargs)
-
+            pred = lambda x: inspect.ismethod(x) or inspect.isfunction(x)
```

Comment:
How come functions were added?  I think the only pan-python parts that are wrapped are methods, not functions.